src/tests/test_middleware.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from django.urls import reverse
+
+
+@pytest.mark.django_db
+def test_removal_of_falsy_query_params(
+    api_client,
+    fetch_auth_token,
+    parkeervakken_schema,
+    parkeervakken_parkeervak_model,
+):
+    """ Prove that falsy query params do not lead to queries."""
+    base_url = reverse("dynamic_api:parkeervakken-parkeervakken-list")
+
+    query_param = ("buurtcode", "A05d")
+    response = api_client.get(base_url + "?{}={}".format(*query_param))
+    assert list(response.wsgi_request.GET.items()) == [query_param]
+
+    query_param = ("buurtcode", "")
+    response = api_client.get(base_url + "?{}={}".format(*query_param))
+    assert list(response.wsgi_request.GET.items()) == []
+
+    response = api_client.get(base_url + "?{}=".format("buurtcode"))
```

Comment:
Dit is hetzelfde als boven ;) 
Dus misshcien `.format` is niet echt nodig.

Reply:
test is er uit, samen met de middleware